components/usage/pkg/apiv1/billing.go

```diff
@@ -67,6 +67,37 @@ func (s *BillingService) UpdateInvoices(ctx context.Context, in *v1.UpdateInvoic
 	return &v1.UpdateInvoicesResponse{}, nil
 }
 
+func (s *BillingService) ReconcileInvoices(ctx context.Context, in *v1.ReconcileInvoicesRequest) (*v1.ReconcileInvoicesResponse, error) {
+	balances, err := db.ListBalance(ctx, s.conn)
+	if err != nil {
+		log.WithError(err).Errorf("Failed to reconcile invoices.")
+		return nil, status.Errorf(codes.Internal, "Failed to reconcile invoices.")
+	}
+
+	creditSummaryForTeams := map[string]stripe.CreditSummary{}
```

Comment:
Can we generalize and talk about attributeId instead of teams?

Reply:
Yes, this requires a couple of changes. For one we need to start setting the `attributionId: <value>` on stripe customers rather than the `teamId` we set now